experiment/autobumper/bumper/bumper.go

```diff
@@ -94,25 +83,54 @@ func (af *fileArrayFlag) Set(value string) error {
 
 // Options is the options for autobumper operations.
 type Options struct {
-	GitHubOrg     string
-	GitHubRepo    string
-	GitHubLogin   string
-	GitHubToken   string
-	GitName       string
-	GitEmail      string
-	RemoteBranch  string
-	OncallAddress string
-
-	BumpProwImages   bool
-	BumpBoskosImages bool
-	BumpTestImages   bool
-	TargetVersion    string
-
-	IncludedConfigPaths fileArrayFlag
-	ExcludedConfigPaths fileArrayFlag
-	ExtraFiles          fileArrayFlag
-
-	SkipPullRequest bool
+	//The GitHub org name where the autobump PR will be created. Must not be empty when SkipPullRequest is false.
+	GitHubOrg string `yaml:"gitHubOrg"`
+	//The GitHub repo name where the autobump PR will be created. Must not be empty when SkipPullRequest is false.
+	GitHubRepo string `yaml:"gitHubRepo"`
+	//The GitHub username to use. If not specified, uses values from the user associated with the access token.
+	GitHubLogin string `yaml:"gitHubLogin"`
+	//The path to the GitHub token file.
+	GitHubToken string `yaml:"gitHubToken"`
+	//The name to use on the git commit. Requires GitEmail. If not specified, uses values from the user associated with the access token
+	GitName string `yaml:"gitName"`
+	//"The email to use on the git commit. Requires GitName. If not specified, uses values from the user associated with the access token.
+	GitEmail string `yaml:"gitEmail"`
+	//The oncall address where we can get the JSON file that stores the current oncall information.
+	OncallAddress string `yaml:"onCallAddress"`
+	//Whether to skip creating the pull request for this bump.
+	SkipPullRequest bool `yaml:"skipPullRequest"`
+	//The URL where upstream images are located. Must not be empty if Target Version is "upstream" or "upstreamStaging"
+	UpstreamURLBase string `yaml:"upstreamURLBase"`
+	//The config paths to be included in this bump, in which only .yaml files will be considered. By default all files are included.
+	IncludedConfigPaths []string `yaml:"includedConfigPaths"`
+	//The config paths to be excluded in this bump, in which only .yaml files will be considered.
+	ExcludedConfigPaths []string `yaml:"excludedConfigPaths"`
+	//The extra non-yaml file to be considered in this bump.
+	ExtraFiles []string `yaml:"extraFiles"`
+	//The target version to bump images version to, which can be one of latest, upstream, upstream-staging and vYYYYMMDD-deadbeef.
+	TargetVersion string `yaml:"targetVersion"`
+	//The name used in the address when creating remote. Format will be git@github.com:{GitLogin}/{RemoteName}.git
+	RemoteName string `yaml:"remoteName"`
+	//List of prefixes that the autobumped is looking for, and other information needed to bump them
+	Prefixes []Prefix `yaml:"prefixes"`
+}
+
+// Prefix is the information needed for each prefix being bumped.
+type Prefix struct {
+	//Name of the tool being bumped
+	Name string `yaml:"name"`
```

Comment:
In the future, never use `yaml` tags, use `json` tags, bceause they are unerstood by both the yaml and the json serializer